src/Microsoft.TestPlatform.CommunicationUtilities/Microsoft.TestPlatform.CommunicationUtilities.csproj

```diff
@@ -19,7 +19,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json">
-      <Version>8.0.3</Version>
+      <Version>9.0.1</Version>
```

Comment:
There was a reason this was made 8.0.3 IIRC. Tagging @mayankbansal018 for context.